File: Quivi.Backoffice.Api/Requests/AcquirerConfigurations/UpsertPaybyrdTerminalAcquirerConfigurationRequest.cs
namespace Quivi.Backoffice.Api.Requests.AcquirerConfigurations
{
    public class UpsertPaybyrdTerminalAcquirerConfigurationRequest : UpsertPaybyrdAcquirerConfigurationRequest
    {
        public string? TerminalId { get; init; }
    }
}

File: quivi.backoffice.react.webapp/src/hooks/api/Dtos/acquirerconfigurations/UpsertPaybyrdTerminalAcquirerConfigurationRequest.ts
import { UpsertAcquirerConfigurationRequest } from "./UpsertAcquirerConfigurationRequest";

export interface UpsertPaybyrdTerminalAcquirerConfigurationRequest extends UpsertAcquirerConfigurationRequest {
    readonly terminalId: string;
    readonly apiKey: string;
}